Servers/UI/OJS.Servers.Ui/ClientApp/src/components/guidelines/trees/Tree.tsx

```diff
@@ -28,6 +29,7 @@ const Tree = ({
     defaultExpanded = [],
     itemFunc,
     treeItemHasTooltip = false,
+    onHandleTreeItemClick,
```

Comment:
Just `onTreeItemClick`.

`on` prefix for components
`handle` prefix for handlers